README.md

```diff
@@ -1,21 +1,24 @@
 # COVIDScenarioPipeline
```

Comment:
Wasn't expecting changes to the Readme.  What's happening here?

Reply:
There was a conflict in the readme that needed to be resolved. The branch originally came from master